src/app/fundraisings/[fundraisingId]/jars/JarsList/AddJarDialog/AddJarDialog.tsx

```diff
@@ -1,8 +1,8 @@
-import { useRef, useState } from 'react';
+import { ReactNode, useContext, useRef, useState } from 'react';
```

Comment:
It's not "AddJar" anymore, right? :) Because this dialog also responsible for editing